src/libcore/macros.rs

```diff
@@ -553,6 +553,64 @@ macro_rules! unimplemented {
     ($($arg:tt)+) => (panic!("not yet implemented: {}", format_args!($($arg)*)));
 }
 
+/// A standardized placeholder for marking unfinished code.
```

Comment:
The unimplemented macro also has this sentence in the docs so you probably want to re-frame that documentation.

Reply:
We can do that only when we stabilize `todo`: https://github.com/rust-lang/rust/pull/56348#issuecomment-442973048

Reply:
aren't macros insta-stable?

Reply:
There are other unstable macros already, like `concat_idents!`

Reply:
@steveklabnik I believe not necessarily. The `dbg!()` macro went through nightly first, and is only now landing on stable.